demo/demo.md

```diff
@@ -215,6 +215,143 @@ Be sure to use the `customColor` attribute on the auro-icon component to allow c
 
 </auro-accordion>
 
+#### Icon only button
+
+Use the `iconOnly` attribute with the `auro-button` element for a style specific to having only an `auro-icon` as the button's content. 
+
+Be sure to also use the `customColor` and `customSize` attributes on the `auro-icon` component to allow colors set in your parent element to pass through to the icon.
+
+<div class="exampleWrapper auro_containedButtons">
+  <auro-button iconOnly>
```

Comment:
Can we update all these examples to include an `aria-label` attribute for voiceover support?